components/apimgt/org.wso2.carbon.apimgt.gateway.extension.listener/src/main/java/org/wso2/carbon/apimgt/gateway/extension/listener/DefaultExtensionListener.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.apimgt.gateway.extension.listener;
+
+import org.wso2.carbon.apimgt.gateway.extension.listener.model.dto.ExtensionResponseDTO;
+import org.wso2.carbon.apimgt.gateway.extension.listener.model.dto.RequestContextDTO;
+import org.wso2.carbon.apimgt.gateway.extension.listener.model.dto.ResponseContextDTO;
+
+/**
+ * Default implementation of ExtensionListener interface.
+ */
+public class DefaultExtensionListener implements ExtensionListener {
```

Comment:
do we need to ship the Default Extenstion

Reply:
removed the default extension